.editorconfig

```diff
@@ -19,3 +19,6 @@ trim_trailing_whitespace = unset
 
 [*.gradle.kts]
 indent_size = 2
+
+[.gitmodules]
+indent_style = tab
```

Comment:
.gitmodule has not been updated in 2 years, but the check is firing (and failing) now for some reason.
Adding an explicit configuration for .gitmodule.